lib/rspec-puppet/support.rb

```diff
@@ -228,9 +228,15 @@ def build_catalog(*args)
     end
 
     def munge_facts(facts)
-      output = {}
-      facts.keys.each { |key| output[key.to_s] = facts[key] }
-      output
+      return facts.reduce({}) do | memo, (k, v)|
```

Comment:
I hope you had a great week off :)

With your feedback, I aligned the munge_facts method to convert all symbols to lowercase strings to align with the current behavior of Puppet and included appropriate test cases.